.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: ruby
 rvm:
+  - 2.4.0
```

Comment:
Sure thing, done. 

I also tried to update the Rubinius version, but Travis is still having a problem loading it.